source/usage.rst

```diff
@@ -0,0 +1,156 @@
+.. _usage:
+
+pymesync - Communicate with a TimeSync API
+==========================================
+
+This module provides an interface to communicate with an implementation of the
+`OSU Open Source Lab`_'s `TimeSync`_ API. An example implementation in Node.js
+can be found `on Github`_.
+
+This module allows users to send times to TimeSync (**send_time()**), get a time
+or a list of times from TimeSync (**get_times()**), and get a project or list of
+projects from TimeSync (**get_projects()**).
+
+Pymesync currently supports the following TimeSync API versions:
+
+* v1
+
+All of these methods return a python list of one to many dictionaries.
+
+* **send_time(baseurl)** - Sends time to baseurl (TimeSync), must include version
```

Comment:
This doesn't look right - send time just takes a time, and baseurl is passed to the constructor, right?